app/views/spree/api/payments/source_views/_stripe.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+attrs = [:id]
+if @current_user_roles.include?("admin")
+  attrs += [:stripe_payment_method_id]
+end
+
+json.call(payment_source, *attrs)
```

Comment:
Similar to the existing one in `solidusio/solidus`:
https://github.com/solidusio/solidus/blob/master/api/app/views/spree/api/payments/source_views/_gateway.json.jbuilder